packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/use-flow.md

```diff
@@ -1,35 +1,38 @@
 
-```bash
-https://${yourOktaDomain}/oauth2/v1/authorize?client_id=0oabucvy
-c38HLL1ef0h7&response_type=code&scope=openid&redirect_uri=https%3A%2F%2Fexample.com&state=state-296bc9a0-a2a2-4a57-be1a-d0e2fd9bb601
+```uri
+https://${yourOktaDomain}/oauth2/v1/authorize?
+   client_id=0oabucvyc38HLL1ef0h7&
+   response_type=code&scope=openid&
+   redirect_uri=https%3A%2F%2Fexample.com&
+   state=state-296bc9a0-a2a2-4a57-be1a-d0e2fd9bb601
 ```
 
 Note the parameters that are being passed:
 
-- `client_id` matches the Client ID of your Okta OAuth application that you created above. You can find it at the bottom of your application's **General** tab.
-- `response_type` is `code`, indicating that we are using the Authorization Code grant type.
-- `scope` is `openid`, which means that the `/token` endpoint returns an ID token. See the **Create Scopes** section of the [Create an authorization server guide](/docs/guides/customize-authz-server/main/#create-scopes).
-- `redirect_uri` is the callback location where the user agent is directed to along with the `code`. This URI must match one of the **Sign-in redirect URIs** that you specified when you created your Okta application in the [Set up your app](#set-up-your-app) section.
-- `state` is an arbitrary alphanumeric string that the authorization server reproduces when redirecting the user agent back to the client. This is used to help prevent cross-site request forgery.
+* `client_id` is the client ID of the app integration that you created earlier. Find it in the Admin Console on your app integration's **General** tab.
+* `response_type` is `code`, indicating that you're using the Authorization Code grant type.
+* `scope` is `openid`, which means that the `/token` endpoint returns an ID token. See the **Create Scopes** section of the [Create an authorization server guide](/docs/guides/customize-authz-server/main/#create-scopes).
+* `redirect_uri` is the callback location where the user agent is directed to along with the `code`. This URI must match one of the **Sign-in redirect URIs** that you specified when you created your app integration earlier.
+* `state` is an arbitrary alphanumeric string that the authorization server reproduces when redirecting the user agent back to the client. This is used to help prevent cross-site request forgery.
 
 See [the OAuth 2.0 API reference](/docs/reference/api/oidc/#authorize) for more information on these parameters.
 
-If the user doesn't have an existing session, making this request opens the Okta sign-in page. If they have an existing session, or after they authenticate, they arrive at the specified `redirect_uri` along with a `code`:
+If the user doesn't have an existing session, this request opens the Okta sign-in page. If they have an existing session, or after they authenticate, they arrive at the specified `redirect_uri` along with a `code`:
 
-```bash
+```uri
 http://localhost:8080/?code=P5I7mdxxdv13_JfXrCSq&state=state-296bc9a0-a2a2-4a57-be1a-d0e2fd9bb601
```

Comment:
```suggestion
https://www.example.com#code=P5I7mdxxdv13_JfXrCSq&state=state-296bc9a0-a2a2-4a57-be1a-d0e2fd9bb601
```
I'm just following the example in the /authorize API response. But you might want to confirm with the Fed team for a current response example.

Reply:
Good spot. It hadn't occurred to me that this might not be an approved example of the response.